libraries/api-extractor/src/IndentedWriter.ts

```diff
@@ -44,8 +65,8 @@ export default class IndentedWriter {
    * Each call to IndentedWriter.increaseIndent() must be followed by a
    * corresponding call to IndentedWriter.decreaseIndent().
    */
-  public increaseIndent(prefix: string = '  '): void {
-    this._indentStack.push(prefix);
+  public increaseIndent(): void {
+    this._indentStack.push(this.spacing);
     this._updateIndentText();
   }
 
```

Comment:
Nevermind, we do have a class, but it is checked into a private project.

---
In reply to: [132347984](https://github.com/Microsoft/web-build-tools/pull/304#discussion_r132347984) [](ancestors = 132347984)